package.json

```diff
@@ -61,6 +61,7 @@
     "react-copy-to-clipboard": "^5.0.2",
     "react-dev-utils": "^10.1.0",
     "react-dom": "^16.9.0",
+    "react-notification-bar": "^1.0.8",
```

Comment:
I don't see any imports to this library, perhaps this was something you were testing out but forgot to remove?